layouts/shortcodes/notices/prototypeShip.html

```diff
@@ -0,0 +1 @@
+{{ partial "notice.html" (merge (dict "page" .Page) (dict "noticeTitle" "Prototype" "noticeDescription" "This Super Capital Ship is classified as a Prototype, a massive, enormously expensive warship with unparalleled firepower and health." "noticeColor" "#ffffff" ) ) }}
```

Comment:
Same as the Premium Ship notice, I feel like `#ffffff` is just kind of a boring color for this. Maybe a purple, just until we can re-add the old gradient?
```suggestion
{{ partial "notice.html" (merge (dict "page" .Page) (dict "noticeTitle" "Prototype" "noticeDescription" "This Super Capital Ship is classified as a Prototype, a massive, enormously expensive warship with unparalleled firepower and health." "noticeColor" "#A408C0" ) ) }}
```
Comparison with the `superCapitalShip` notice:
![image](https://github.com/user-attachments/assets/8582f7da-e4a8-479a-98f6-75e37a019dfa)